gitops/overlays/dev/configs/frontend/config.conf

```diff
@@ -11,7 +11,7 @@ OTEL_TRACES_ENDPOINT=https://dynatrace.dev-dp.admin.dts-stn.com/e/21a07aef-852b-
 #
 # Application feature flags configuration
 #
-ENABLED_FEATURES=doc-upload,email-alerts,hcaptcha,view-personal-info,view-applications,view-letters,view-messages,edit-personal-info,status,authenticated-status-check,show-prototype-banner
+ENABLED_FEATURES=doc-upload,email-alerts,hcaptcha,view-personal-info,view-applications,view-letters,view-messages,edit-personal-info,status,authenticated-status-check,show-prototype-banner, update-governmental-benefit
```

Comment:
@gregory-j-baker told me to create new PRs for changes to the gitops sub-project so I will pay it forward.

Reply:
Removed: 3e3028732d9cee2c8c61af92f40a0aa376b10515.